vars/main.yml

```diff
@@ -1,11 +0,0 @@
----
-
-# Default WATO host attributes. This variable, when being merged with
-# `checkmk_agent__host_attributes`, allows to successfully compare the WebAPI
-# get_host() response in case the `contactgroups` key is set.
```

Comment:
This is mainly done for transparency reasons. `checkmk_agent__default_host_attributes` is laster combined with `checkmk_agent__host_attributes`. Ideally, the `checkmk_agent__fact_host_attributes` could also be moved to defaults. Would you be ok with this or should I change it back to vars?